src/foam/nanos/cron/services

```diff
@@ -3,12 +3,13 @@ p({
   "name":"cronDAO",
   "serve":true,
   "serviceScript":"""
+    mdao = new foam.dao.MDAO(foam.nanos.cron.Cron.getOwnClassInfo());
     return new foam.dao.EasyDAO.Builder(x)
       .setOf(foam.nanos.cron.Cron.getOwnClassInfo())
+      .setMdao(mdao)
+      .setInnerDAO(new net.nanopay.security.HashingJDAO(x, "SHA-256", true, false, mdao, "cronjobs"))
```

Comment:
net.nanopay.security.HashingJDAO is from nanopay side.

Reply:
crap crap crap.  we need to use hashing dao where we may have the need to support instance specific scripts, crons. ...  i have having to duplicate on the nanopay side for this stuff. ok.

Reply:
Replaced with JDAO.  I'll duplicate service entries on the nanopay side.